instrumentation/otel-webserver-module/src/core/sdkwrapper/SdkHelperFactory.cpp

```diff
@@ -96,9 +96,17 @@ SdkHelperFactory::SdkHelperFactory(
         " and LibraryVersion " << libraryVersion);
 
     // Adding trace propagator
-    using MapHttpTraceCtx = opentelemetry::trace::propagation::HttpTraceContext;
-    mPropagators.push_back(
-        std::unique_ptr<MapHttpTraceCtx>(new MapHttpTraceCtx()));
+    if(config->getOtelPropagatorType() == "b3"){
```

Comment:
Also, since this variable is set by user as string, there can be typo in such case. How are we handling the case when the variable is set as "B3" ? Where are we defaulting to when user set to some random characters?

Reply:
We are not handling for "B3", it needs to "b3". When the entered string is not "b3", the if else statement will go for W3C headers.